src/main/java/com/microsoft/azure/spring/data/cosmosdb/Constants.java

```diff
@@ -29,8 +28,11 @@ private Constants() {
 
     public static final String OBJECTMAPPER_BEAN_NAME = "cosmosdbObjectMapper";
 
+    public static final String SQL_KEYWORD_NULL = "";
```

Comment:
sql keyword should be in separate class, better be enum.

Reply:
Got one enum for this `CriteriaType`, for each mapping to one String of Keyword, as it only used once, it may migrate to that enum.